packages/tag/src/components/Tag.story.js

```diff
@@ -3,35 +3,35 @@ import { storiesOf } from "@storybook/react";
 import { withInfo } from "@storybook/addon-info";
 import { action } from "@storybook/addon-actions";
 
-import RemovableTags from "./RemovableTags";
+import Tag from "./Tag";
 
 storiesOf("Removable-tags", "module")
   .add(
     "Default",
     withInfo()(() => (
       <div style={{ marginTop: "20px" }}>
-        <RemovableTags onRemove={action("tag removed")} text="THIS IS A TAG" />
+        <Tag onRemove={action("tag removed")} text="THIS IS A TAG" />
```

Comment:
Refactor to `<Tag onRemove={action("onRemove")} text="THIS IS A TAG" />`
This way we can see which function was called on the action logger